docs/csharp/language-reference/tokens/toc.md

```diff
@@ -0,0 +1,3 @@
+# [C# Special Characters](index.md)
+## [$](interpolated.md)
```

Comment:
Suggestion: Should we spell out more in the TOC? For example, '$ -- string interpolation' and '@ -- verbatim string'? I'm not sure which readers would be more likely to search for.